crowbar_framework/config/repos.yml

```diff
@@ -5,169 +5,121 @@ suse-12.0:
       name: "SLES12-Pool"
       required: "mandatory"
       product_name: "common"
-      content_key:
```

Comment:
These changes are totally different to the strongly pushed schema defined within https://etherpad.nue.suse.com/p/cloud-repo-checks at "Repository schema"